packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-details.controller.js

```diff
@@ -0,0 +1,24 @@
+import { getDataProcessingUiUrl } from '../data-processing.utils';
+import {
+  DATA_PROCESSING_STATUS_TO_CLASS,
+  DATA_PROCESSING_GUIDE_URL,
+} from '../data-processing.constants';
+
+export default class {
+  /* @ngInject */
+  constructor(
+    $state,
+    $uibModal,
```

Comment:
Are those injections really needed ?

Reply:
You're right! Not needed! Could be removed.